Kubernetes/HOWTO-on-prem.md

```diff
@@ -59,41 +78,41 @@ In the `manifest` folder, run the Python script, passing your master IP and the
 This will generate a set of YAML files. You should [re]move the Python script so that K8s doesn't mistake it for a manifest and cause problems.
 
 ### Configure & Run Kubernetes ###
-Run the script `configure-kubectl.sh`, passing your external IP again, to create a configuration in `~/.kube/config` containing all of the certificate data and other parameters:
+Run the script `configure-kubectl.sh` passing your external IP again. This will create a configuration in `~/.kube/config` containing all of the certificate data and other parameters:
 
     ~/kube $ ./configure-kubectl.sh 10.123.45.67
 
-Run the script `start-kubelet.sh`. You may need to run it with `sudo`. Then, in another shell, copy the Kubernetes configuration file to the local kubelet mount path:
+Run the script `start-kubelet.sh`. You may need to run it with `sudo`. Then, in another shell, watch your working directory (`~/kube`) until the folder `kubelet/` appears. Immediately copy the Kubernetes configuration file to it:
 
-    $ sudo cp ~/.kube/config kubelet/
+    ~/kube $ sudo cp ~/.kube/config kubelet/
 
 **TODO**: Why is this necessary? Shouldn't this be done automatically by one of the containers?
 
 In another terminal session, run the Kubeproxy script, passing your cluster CIDR (you may also need to run this one with `sudo`):
 
     ~/kube $ ./kubeproxy.sh 192.168
 
-After some time, you should see the following system state:
+After a few minutes, you should see the following system state:
 
   - Under `docker ps`, you should see worker and pod containers mirroring the configs in `manifest/` and in `addons/`.
-  - A call to `~/kube/bin/kubectl cluster-info` should show info for the Kubernetes master, DNS, and heapster.
-  - A new interface `cbr0` under `ipconfig`, with your chosen cluster CIDR.
+  - A call to `~/kube/bin/kubectl cluster-info` should show info for the Kubernetes master API server, plus the DNS and heapster addons.
+  - A new interface `cbr0` under `ifconfig`, with your chosen cluster CIDR.
 
 ## Join a Windows Node ##

```

Comment:
TODO: We should add instructions for adding a Linux node as well (or point to link in K8s documentation)